tools/cldr-apps/src/main/liberty/config/server.xml

```diff
@@ -1,15 +1,17 @@
 <server description="cldr-apps Liberty server">
 	<!-- tag::featureManager[] -->
 	<featureManager>
-		<feature>jsonb-1.0</feature>
+		<feature>jsonb-1.0</feature> <!-- 2.0 -->
```

Comment:
what do the comments indicate?

Reply:
Oops, typo.  Those are the 'latest i could find' trying to track down a version conflict.  (I wanted to make this unobtrusive)

Reply:
I've removed the comments.